guidance/models/_model.py

```diff
@@ -38,17 +42,17 @@
 
 class Tokenizer:
     '''This is the standardized tokenizer interface used by guidance models.
-    
+
```

Comment:
the space deletions are a huge number of diffs here, but I think I'm OK merging these for now since we're hoping to (in the next few days) lint the codebase and start enforcing auto-linting. Just trying to get through PR merges first to not cause merge headaches for everyone :)

Reply:
Damn, I tried to prevent this. I'll fix it.

Reply:
> we're hoping to (in the next few days) lint the codebase and start enforcing auto-linting.

I'm very happy to hear this

Reply:
> Damn, I tried to prevent this. I'll fix it.

On second thoughts, if it's not _too_ much of a pain, minimizing the diff space here would be much appreciated :)

Reply:
Resolved

Reply:
> we're hoping to (in the next few days) lint the codebase and start enforcing auto-linting.

I'd love to be @ mentioned on this PR